ci/test/notebooks.sh

```diff
@@ -6,9 +6,14 @@ set -o pipefail
 export LIBCUDF_KERNEL_CACHE_PATH=${WORKSPACE}/.jitcache
 
 source /opt/conda/bin/activate rapids
+
+# PyTorch is intentionally excluded from our Docker images due
+# to its size, but some notebooks still depend on it.
+conda install -y -c pytorch pytorch
```

Comment:
I've been purposely holding off on pytorch for a couple reasons.

- Pytorch doesn't support CUDA 11 quite yet I don't think
- You have to remove packages from our environment to make this, I'm not sure what other adverse effects this may have

See: https://github.com/rapidsai/cusignal/blob/branch-0.16/ci/gpu/build.sh#L88